### ISSUE TITLE
usedrange expands to the maximum when DeleteColumn

### ISSUE THREAD

**Issue by ninoxscutulata** [EPPlusSoftware/EPPlus#621]
EPPLUS 5.8.6

Change in the number of usedrange columns when deleting a column using epplus

step-1. before delete, usedrange columns count is 3
step-2. Delete column and save using epplus 5.8.6
step-3. open excel file using ms-excel
step-4. usedrange columns count is 16381 and horizontal scrollbar expanded maximize

Additionally, as a result of testing using closedxml , it was normal. 
There seems to be an error in epplus handling 'some' xlsx files. not all.

This issue can be reproduced using the xlsx file and code attached below

[test xlsx file](https://drive.google.com/file/d/10J-LWsemJl_6f01ssmxULiXaQbRuB6CQ/view?usp=sharing)

```
ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
string xlpath = "test.xlsx";
var workbookFileInfo = new FileInfo(xlpath);
using (var excelPackage = new ExcelPackage(workbookFileInfo))
{
  var worksheet = excelPackage.Workbook.Worksheets.FirstOrDefault();
  worksheet.DeleteColumn(1, 3);
  excelPackage.Save();
}
```

**Comment by JanKallman** [comment]
Can you attach the actual test.xlsx file instead of the image?

**Comment by JanKallman** [comment]
This happens due to the last column element in the list below. 
```xml
<cols>
<col customWidth="1" bestFit="1" style="1" width="9.5" max="1" min="1"/>
<col customWidth="1" style="1" width="68" max="2" min="2"/>
<col customWidth="1" style="3" width="8.5" max="3" min="3"/>
<col style="1" width="9" max="16384" min="4"/>
</cols>
```
With EPPlus you get:
`<col style="1" customWidth="1" width="9" max="16381" min="1"/>`
EPPlus subtracts three from the col element when it is max, which seems to be a different behavior than Excel. I'll have a look at it.

**Comment by JanKallman** [comment]
Fixed in 5.8.9
